tests/storages_tests/test_storages.py

```diff
@@ -1190,6 +1190,8 @@ def test_retry_failed_trial_callback(storage_mode: str, max_retry: Optional[int]
         study = optuna.create_study(storage=storage)
 
         trial = study.ask()
+        _ = trial.suggest_uniform("_", -1, -1)
```

Comment:
A nit, but can we omit `_ = ` if we do not use the return value?
In addition, please use `suggest_float` instead of `suggest_uniform`.
c.f., https://github.com/optuna/optuna/pull/3027.
```suggestion
        trial.suggest_float("_", -1, -1)
```